### DISCUSSION TITLE
docker stop sliently

### DISCUSSION THREAD

**Discussion by welegent2010** [TriliumNext/Trilium#4348] [General]
Does annbody may find container manager ,(docker before), run for hours, and stopped no reason. So every time when I need sync, I will restrart docker trilium  server again.

**Comment by zerebos** [comment]
You can add `restart: unless_stopped` to your docker compose file

**Comment by jwhonce** [comment]
If it helps, I run the image from docker hub under quadlet/podman on a fedora server both versions 0.60.4 and the 0.61 betas without them crashing. Does `docker logs` show anything useful?

I love that @zadam makes the container and flatpack images available.  It makes using this great software so easy. I know this is apples and oranges but here is a configuration that works.

**Comment by welegent2010** [comment]
good works. I `m used to use trilium to record my notes on my computer. But cannot sync with my nas. It some difficult for me to add&nbsp;unless_stopped&nbsp;to your docker compose file, I`m newer in my nas. So I think if you can add it in next upgrade.&nbsp;
thanks a lot.


PIZZA&nbsp;HUT
***@***.***



&nbsp;





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zadam/trilium"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年10月24日(星期二) 中午11:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zadam/trilium] docker stop sliently (Discussion #4348)





 
You can add restart: unless_stopped to your docker compose file
